test/evergreen.yml

```diff
@@ -713,21 +692,12 @@ tasks:
       - func: "make check all"
 
   - name: make-check-msan-test
-    depends_on:
-      - name: compile-msan
     commands:
-      - func: "fetch artifacts"
-        vars:
-          dependent_task: compile-msan
+      - func: "get project"
       - func: "compile wiredtiger"
-        vars:
-          # We don't compile C++ for memory sanitized testing as it creates false positives
-          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=skip PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-fsanitize=memory -ggdb"
-          posix_configure_flags: --enable-silent-rules --enable-strict --enable-diagnostic --disable-static
       - func: "make check all"
         vars:
           smp_command: -j 1
```

Comment:
It looks these 2 lines are the only differences compared to `make-check-test` task. Could not remember exactly why MSAN test requires a single job (avoiding parallelization). If it's not necessary we could recycle this task.

Reply:
The `-j 1` was added in the fix for WT-5693. I've removed `make-check-msan-test` and instead excluded the problematic test from being run.